### ISSUE TITLE
error message in Analyze

### ISSUE THREAD

**Issue by dburin** [neuropsychology/psycho.R#58]
Hello! Before posting my request, many thanks for the package!
I have used it succesfully with simpler models. But with glmer I get the following error message:
"Error in glmer(formula = ....._ : 
  fitting model with the observation-level random effect term failed. Add the term manually
In addition: Warning message:
In KhatriRao(sm, t(mm)) : (p <- ncol(X)) == ncol(Y) is not TRUE"

[for the simpler model: one continuous predictor, one categorical predictor, one random term; standard glmer works fine]

Again, many thanks for your work!

**Comment by DominiqueMakowski** [comment]
@dburin Wow, this error message doesn't look like anything to me.

Does a normal summary() directly on the model works? 

Could you attach a reproducible example with a minimal dataset required to reproduce this error? So I can try to understand where the error origins from. Thanks

**Comment by dburin** [comment]
Yes, the summary reads:
> todo17.glmer5 <- glmer(comprension ~ ZAptVerbal+Hsonline+(1|Items), family = binomial)
> summary(todo17.glmer5)
Generalized linear mixed model fit by maximum likelihood (Laplace Approximation) ['glmerMod']
 Family: binomial  ( logit )
Formula: comprension ~ ZAptVerbal + Hsonline + (1 | Items)

     AIC      BIC   logLik deviance df.resid 
  4836.0   4874.3  -2412.0   4824.0     4379 

Scaled residuals: 
    Min      1Q  Median      3Q     Max 
-3.9109 -0.7400  0.3939  0.6255  4.1358 

Random effects:
 Groups   Name        Variance Std.Dev.
 Items (Intercept) 1.362    1.167   
Number of obs: 4385, groups:  Items, 20

Fixed effects:
            Estimate Std. Error z value Pr(>|z|)    
(Intercept)   0.2819     0.2730   1.033 0.301724    
ZAptVerbal    0.3068     0.0372   8.247  < 2e-16 ***
Hsonline2     0.2280     0.1029   2.215 0.026765 *  
Hsonline3     0.2440     0.1096   2.226 0.026007 *  
Hsonline4     0.4021     0.1044   3.851 0.000118 ***
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

Correlation of Fixed Effects:
           (Intr) ZAptVr Hsnln2 Hsnln3
ZAptVerbal  0.048                     
Hsonline2  -0.222 -0.083              
Hsonline3  -0.212 -0.155  0.558       
Hsonline4  -0.223 -0.159  0.586  0.563

I am rather new to this, I am sure it is something **I** did wrong but would not know what. Perhaps in variable class? I checked, "zAptVerbal" is numeric, "Hsonline" is a factor (4 levels), and "Items" is character (20 different items).
